PyPi_Setup.md

```diff
@@ -0,0 +1,190 @@
+# PyPi setup 
+
+## 1. Introduction
+
+This documentation provides a comprehensive guide on how to set up and deploy a Python package to the Python Package Index (PyPI). The package is designed to facilitate the sharing of Python projects, enabling users to install it easily using pip.
+
+### Purpose
+
+The objectives of this guide are to help users:
```

Comment:
Can you change this to first person, idk is sounds a little wired while mentoning users in the docs, it should be something like 
```suggestion
The objectives of this guide is to help you with:
```